portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,30 @@
   </head>
   <body>
     <div id="about-content">
-        <div id = "about">
-            <div id="nav-bar">
-                <a href ="#about">about</a>
-                <a href ="#suggestions">suggestions</a>
-                <a href ="#map">map</a>
+        <div id="nav-bar">
+            <a href ="#about">about</a>
+            <a href ="#suggestions">suggestions</a>
+            <a href ="#map">map</a>
+        </div>
+        <div id = "intro">
+            <div id = "intro-txt">
+                <div id = "intro-txt-header"><span id = "highlight">hello</span></div>
+                <div id = "intro-txt-paragraph">I'm <span id = "highlight">Sarah</span>, an aspiring software developer</div>
+            </div>
+            <img src="images/devices-img.jpg" id = "intro-img"/>
+        </div>
+        <div id = "about-me-section">
+            <div id = "background-info-section">
+                <img src = "images/headshot.jpg" id = "headshot-img"/>
+                <p id = "background-txt">I am originally from Clarksburg, Maryland but I am currently a <span id = highlight>first generation</span> student 
+                    at the <span id = highlight>University of Michigan</span> (<span id =  "go-blue">Go Blue!</span>). This summer I am working on a web application 
```

Comment:
Nitpick: Can you quote the id name here? Eg id="highlight"

https://google.github.io/styleguide/htmlcssguide.html#HTML_Quotation_Marks

Reply:
fixed